analyzer/consensus/consensus.go

```diff
@@ -249,7 +254,7 @@ func (m *Main) prepareBlockData(ctx context.Context, height int64, batch *storag
 		return err
 	}
 
-	for _, f := range []func(*storage.QueryBatch, *storage.BlockData) error{
+	for _, f := range []func(*storage.QueryBatch, *storage.ConsensusBlockData) error{
```

Comment:
We do this for better/scoped naming?

Reply:
Partially. More specifically because the type for [consensus blocks](https://pkg.go.dev/github.com/oasisprotocol/oasis-core/go/consensus/api#Block) differs from that for [runtime blocks](https://pkg.go.dev/github.com/oasisprotocol/oasis-core/go/roothash/api/block#Header).

Consensus blocks may contain consensus-backend specific metadata, and runtime blocks contain additional merkle roots for storage, etc.

Reply:
Yup, that's correct.